main/tests/test_views.py

```diff
@@ -0,0 +1,128 @@
+"""Tests for views are at this file."""
+from django.test import TestCase, Client
+from django.contrib.auth.models import User
+from django.core.urlresolvers import reverse
+from django.utils import timezone
+
+from main.models import Post, Comment
+
+
+class ViewsTest(TestCase):
+    """Testing class for views."""
+
+    def setUp(self):
+        """Prepare data for testing."""
+        self.USERNAME = 'admin'
```

Comment:
Feel free to move this into the class level (as a property) or [`setUpClass`](https://docs.python.org/3/library/unittest.html#unittest.TestCase.setUpClass), since you don't need to reset it after each test run.